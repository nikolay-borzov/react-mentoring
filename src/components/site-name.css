.site-name {
  color: var(--color-primary);
  font-weight: 700;
}