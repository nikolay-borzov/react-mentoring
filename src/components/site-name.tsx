import React from 'react'
import styled from 'styled-components'

const SiteNameStyled = styled.span`
  color: var(--color-primary);
  font-weight: 700;
`

export const SiteName: React.FunctionComponent = () => <SiteNameStyled>Movie Search</SiteNameStyled>
