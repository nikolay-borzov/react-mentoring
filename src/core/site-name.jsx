import React from 'react'

export function SiteName() {
  return <span className="site-name">Movie Search</span>
}
