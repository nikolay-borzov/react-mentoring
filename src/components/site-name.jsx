import React from 'react'

import './site-name.css'

export function SiteName() {
  return <span className="site-name">Movie Search</span>
}
