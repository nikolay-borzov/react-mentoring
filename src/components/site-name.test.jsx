import React from 'react'
import renderer from 'react-test-renderer'

import { SiteName } from './site-name'

describe('SiteName component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SiteName> Site Name </SiteName>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
