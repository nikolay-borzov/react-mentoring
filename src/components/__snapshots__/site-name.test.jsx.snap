// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteName component renders correctly 1`] = `
<span
  className="site-name"
>
  Movie Search
</span>
`;
